### PR TITLE
Option Page Typos

### PR DESCRIPTION
**Pull request by andymaheshw**
#### Problem description
There are typos on this page and one confusing sentence. Trying to learn the workflow.

### CODE REVIEW

**Comment by TomAugspurger** [comment]
@andymaheshw thanks! keep em coming!

File: doc/source/options.rst [hunk 1]
```diff
@@ -28,7 +28,7 @@ You can get/set options directly as attributes of the top-level ``options`` attr
    pd.options.display.max_rows = 999
    pd.options.display.max_rows
 
-There is also an API composed of 5 relevant functions, available directly from the ``pandas``
+The API is composed of 5 relevant functions, available directly from the ``pandas``
```
Comment (jorisvandenbossche) [inline doc/source/options.rst]: The "there is also" does give a bit more the notion of "next to the possibility of directly getting/setting options on`pd.options. ..`, you also have some functions to work with them". So maybe "Additionally, the ..", or "the api is further composed ..." ?

**Review by jorisvandenbossche — COMMENTED** [review]
small comment, for the rest looks good

File: doc/source/options.rst (hunk 1 above)
Comment (jorisvandenbossche) [inline doc/source/options.rst]: OK, ignore this :-) as Tom already merged (not that important)

**Pull request merged**